sopel/modules/instagram.py

```diff
@@ -39,11 +39,20 @@ def instaparse(bot, trigger, match):
     try:
         bot.say(parse_insta_json(json))
     except ParseError:
-        LOGGER.exception(
-            "Unable to find Instagram's payload for URL %s", instagram_url)
-        bot.say(
-            "Unable to parse this Instagram URL. "
-            "It's probably a temporary error; try again in a bit.")
+        try:
+            json = get_oembed_json(instagram_url)
+            bot.say(parse_oembed_json(json))
+        except ParseError:
+            LOGGER.exception(
+                "Unable to find Instagram's payload for URL %s", instagram_url)
+            bot.say(
+                "Unable to parse this Instagram URL. "
+                "It's probably a temporary error; try again in a bit.")
```

Comment:
```suggestion
                "It's probably a temporary error; try again in a bit.")
        else:
            LOGGER.info(
                "Used fallback to oEmbed metadata; "
                "Sopel's IP may be blocked by Instagram.")
```

Reply:
This I like and will implement, commit and rebase, and force push.